gui/wxpython/core/toolboxes.py

```diff
@@ -789,7 +789,10 @@ def new_translator(string):
     sys.displayhook = new_displayhook
     sys.__displayhook__ = new_displayhook
 
-    import __builtin__
+    if sys.version_info[0] == 2:
+        import __builtin__
+    elif locals().get("__builtin__", globals().get("__builtin__")) is None:
+        import builtins as __builtin__
 
     __builtin__._ = new_translator
```

Comment:
Just drop the Python 2 part from here. The rest can be simpler, `python/grass/init.py` uses:

```python
import builtins as _builtins
_builtins.__dict__["_"] = _translate
```

Reply:
I droped all Python 2 from that test now...